uint16  MAX_ANGLE = 40
float32 f1_angle_in_radians
float32 f2_angle_in_radians
float32 f3_angle_in_radians
float32 f4_angle_in_radians

